flake8_quotes/__init__.py

```diff
@@ -10,33 +10,57 @@ class QuoteChecker(object):
     version = __version__
 
     QUOTES = {
-        # When user wants only single quotes
+        # Allow single quotes
+        # Fail on double quotes
```

Comment:
This comment is redundant to the content, the old comment stated the scenario when the values were used =/